server/app/lib/crm.js

```diff
@@ -0,0 +1,75 @@
+const {ov_config} = require("./ov_config")
+const axios = require("axios")
+
+async function getAmbassadorHSID(email) {
+  let hs_key = ov_config.hubspot_api_key
+
+  console.log(`[HS] Calling API with: ${email}`)
+
+  try {
+    let response
+    if (hs_key) {
```

Comment:
can this come up above the try, for an early fail if missing hs_key?

Reply:
updated